SparkPost/matchbox
# UX-165 Add padding and maxWidth system props to modal

### What Changed
- Adds `padding` and `maxWidth` system props to modal with defaults


### How To Test or Verify
- Run storybook with `npm run start:storybook`
- Verify max width and padding properties work

### PR Checklist
